keanu-project/src/main/java/io/improbable/keanu/network/ProtobufLoader.java

```diff
@@ -213,13 +205,39 @@ private Vertex instantiateVertex(Class vertexClass,
         }
     }
 
+    private Object getParameter(Parameter methodParameter,
+                                Map<String, Object> paramMap,
+                                KeanuSavedBayesNet.Vertex vertex) {
+        LoadVertexParam paramAnnotation;
+        LoadShape shapeAnnotation;
+
+        if ((paramAnnotation = methodParameter.getAnnotation(LoadVertexParam.class)) != null) {
+            Object parameter = paramMap.get(paramAnnotation.value());
+            if (parameter == null) {
+                throw new IllegalArgumentException("Failed to create vertex due to missing parent: "
+                    + paramAnnotation.value());
+            }
+            return parameter;
+        } else if ((shapeAnnotation = methodParameter.getAnnotation(LoadShape.class)) != null) {
```

Comment:
`shapeAnnotation` is never used

Reply:
Yeah - I just added it for consistency with the paramAnnotation case.  I guess it's not required - I'll remove.